docker/server/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "Applying database migrations"
+python manage.py migrate --noinput
+
+echo "Starting server"
+python manage.py runserver 0.0.0.0:8081
```

Comment:
probably later on, but this wont work for production

Reply:
Yup, have not configured WSGI/ ASGI